src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -21,142 +21,153 @@ signupBanner:
+<img
+  title="enable distributed tracing"
+  alt="enable distributed tracing"
+  src={browserInstallStep3}
+  style={{ maxWidth: '70%' }}
+/>
+</Step>
 
-For more information, see the instructions for your APM agent:
+<Step>
+### Name your browser app
 
-* [Go](/docs/agents/go-agent/features/install-new-relic-browser-go-apps)
-* [Java](/docs/agents/java-agent/instrumentation/page-load-timing-java)
-* [.NET](/docs/agents/net-agent/features/page-load-timing-net)
-* [Node.js](/docs/agents/nodejs-agent/supported-features/page-load-timing-nodejs)
-* [PHP](/docs/agents/php-agent/features/page-load-timing-php)
-* [Python](/docs/agents/python-agent/supported-features/page-load-timing-python)
-* [Ruby](/docs/agents/ruby-agent/features/page-load-timing-ruby)
+When selecting a name for your browser app, you've got two options: 
 
-### Create a browser app with NerdGraph API [#create-browser]
+* Select an existing app already monitored by an APM agent. 
+* Create a new name for a standalone browser app. 
 
-Use the NerdGraph API to create a browser app manually. See this [Nerdgraph tutorial](/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph#create-browser). 
+Choose one of these options and click **Enable**. 
 
-<InstallFeedback />
+<img
+  title="name or connect an existing browser app"
+  alt="name or connect an existing browser app"
+  src={browserInstallStep4}
+  style={{ maxWidth: '70%' }}
+/>
+</Step>
 
-## Browser agent types: Lite, Pro, Pro+SPA [#agent-types]
+<Step>
+### (For copy/paste method only) Inject the JavaScript snippet
 
-We have three types of browser agents: Lite, Pro, and Pro+SPA. The agent type has no impact on your billing.
+If you selected the copy/paste option, it's time to add the JavaScript snippet to each of your browser app webpages' HTML. 
 
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "150px" }}>
-        Browser agent type
-      </th>
+In the UI, copy the JavaScript snippet that's been customized for your app instrumentation, and paste it into the `head` of each of your wedpages. Make sure to paste it as close to the top of the `head` as possible, but after any position-sensitive META tags (such as X-UA-Compatible and charset). For more information on the snippet placement, see [JavaScript placement requirements](/docs/browser/new-relic-browser/page-load-timing-resources/instrumentation-browser-monitoring#javascript-placement)
 
-      <th>
-        Comparison
-      </th>
-    </tr>
-  </thead>
+<img
+  title="copy and paste the js snippet"
+  alt="copy and paste the js snippet from the UI"
+  src={browserInstallStep5}
+  style={{ maxWidth: '70%' }}
+/>
 
-  <tbody>
-    <tr>
-      <td>
-        Pro+SPA
-      </td>
+<Callout variant="tip">
```

Comment:
Should this APM list more properly go above in the choose deployment method section?

Maybe it's worth sticking it in a collapser?